doc/ref_arch/openstack/README.md

```diff
@@ -8,7 +8,7 @@ This is OpenStack based Reference Architecture (RA-1)
 
 **Version: _0_**
 
-**Release Date: _1st July 2021_**
+**Release Date: _7th December 2021_**
```

Comment:
Example according to https://wiki.anuket.io/display/HOME/Lakelse+Schedule

```suggestion
**Release Date: _4th January 2022_**
```

Reply:
done